README.md

```diff
@@ -93,6 +93,49 @@ It is also possible for you to create a simple file upload button which yields t
 {{/file-upload}}
 ```
 
+The examples above using angle bracket invocation, available in Ember 3.4+
+
+```handlebars
+<FileDropzone @name={{"photos"}} as |dropzone queue|>
```

Comment:
Strings do not need {{}}, so `@name="photos"` should be used instead.

Reply:
I swear I saw examples showing curlies on strings - can't find it now - corrected